starwars/src/App.js

```diff
@@ -1,41 +1,74 @@
 import React, { Component } from 'react';
-import './App.css';
+import './App.css';	import './App.css';
+import Card from './components/Cards.js';
+import Pagenavigation from './components/Pagenavigation.js';
 
+
+ function next (state, props) {
+  return {
+   url: state.nextUrl
+  };
+};
+
+ function prev (state, props) {
```

Comment:
remember what we talked about: these functions (`next` and `prev`) are not part of the MVP, they are part of the stretch goals